### PR TITLE
Extract Byte2 machine

### PR DESCRIPTION
**Pull request by Schaeff**
requires #1513

### CODE REVIEW

File: importer/src/path_canonicalizer.rs [hunk 1]
```diff
@@ -140,7 +140,32 @@ impl<'a> Folder for Canonicalizer<'a> {
                         canonicalize_inside_expression(e, &self.path, self.paths);
                     }
                 }
-                _ => {}
+                MachineStatement::InstructionDeclaration(_, _, i) => {
```
Comment (chriseth) [inline importer/src/path_canonicalizer.rs:143]: Can we implement `Children<E>` for Instruction?

File: std/machines/arith.asm [hunk 2]
```diff
@@ -10,6 +10,19 @@ use std::convert::expr;
 use std::prover::eval;
 use std::prover::Query;
 
+// A machine to check that a field element represents two bytes. It uses an exhaustive lookup table.
```
Comment (chriseth) [inline std/machines/arith.asm:13]:
```suggestion
/// A machine to check that a field element represents two bytes. It uses an exhaustive lookup table.
```

**Review by leonardoalt — COMMENTED** [review]
Could go to std instead?

**Comment by Schaeff** [comment]
> Could go to std instead?

done

**Review by georgwiese — APPROVED** [review]
LGTM

**Pull request merged**